src/camel/camel-debug.c:1078:3: error: expected expression
                gboolean found = FALSE;

Index: src/camel/camel-debug.c
--- src/camel/camel-debug.c.orig
+++ src/camel/camel-debug.c
@@ -1073,8 +1073,8 @@ backtrace_matches (const Backtrace *match_bt,
 	flink = find_bt->lines;
 	mlink = NULL;
 	lt = lines_tolerance;
-	do {
   try_next_frame:
+	do {
 		gboolean found = FALSE;
 		BacktraceLine *fline = flink->data;
 
